android/app/build.gradle

```diff
@@ -48,13 +50,37 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
+    signingConfigs {
+        release {
+            storeFile file(System.getenv("HOME") + "/keystores/allthenews.keystore")
+            storePassword System.getenv("BITRISEIO_ANDROID_KEYSTORE_PASSWORD")
+            keyAlias System.getenv("BITRISEIO_ANDROID_KEYSTORE_ALIAS")
+            keyPassword System.getenv("BITRISEIO_ANDROID_KEYSTORE_PRIVATE_KEY_PASSWORD")
+        }
+    }
+
     buildTypes {
         release {
-            // TODO: Add your own signing config for the release build.
-            // Signing with the debug keys for now, so `flutter run --release` works.
-            signingConfig signingConfigs.debug
+            signingConfig signingConfigs.release.storeFile.exists() ? signingConfigs.release : signingConfigs.debug
```

Comment:
wydaje mi się, że mimo wszystko nie powinien lecieć callback na signingConfigs.debug

Reply:
dlaczego? jeżeli nie mamy klucza releasowego, czyli wtedy kiedy budujemy lokalnie to podpisujemy kluczem debugowym. Jak budujemy na bitrise i mamy klucz releasowy to podpisujemy kluczem releasowym

Reply:
chwilę pomyślałem i przyznaję rację ;)